docker/README.md

```diff
@@ -23,32 +65,46 @@ common use-cases to get you started quickly.
 
 Choose this method if you do not want to setup your own Supabase instance, but
 only launch StudyU applications with docker. This might be useful if you want to
-run StudyU after you made changes to the codebase. Switch to the `.env.local`
-[environment file](https://github.com/hpi-studyu/studyu/blob/dev/flutter_common/lib/envs/.env.local)
-to use the StudyU database with local StudyU applications. For this modify the
-`docker-compose-*.yml` files you want to run (see below), and replace
-`../flutter_common/lib/envs/.env.selfhost` with
-`../flutter_common/lib/envs/.env.local` in the section `env_file`.
+run StudyU after you made changes to the codebase.
+
+You can either use the StudyU CLI or manually run the docker-compose commands.
+
+### StudyU CLI
+
+Run the StudyU CLI with `./studyu-cli.sh config` and choose the following packages:
+
+- `studyu-app`
+- `studyu-designer`
+
+Then run `./studyu-cli.sh start` to start the StudyU App and the StudyU Designer.
+
+Read on at [Futher customize the setup](#further-customize-the-setup) to learn
+how to change the default database and ports.
+
+### Manual Docker Compose steps
 
 To start both the StudyU App and the StudyU Designer, simply run `docker compose
-up`.
+-f studyu/docker-compose.yml up`.
 
 If you only want to start a single StudyU application run `docker compose -f
-docker-compose-{module} up`. Make sure to replace `{module}` with one of the
-following:
+studyu/docker-compose-{module} up`. Make sure to replace `{module}` with one of
+the following:
 
 - `app`: Start the StudyU App
 - `designer`: Start the StudyU Designer
 - `designer_legacy`: Start the outdated Designer v1 (not recommended)
 
+### Further customize the setup
+
 The default ports are 8080 for the [StudyU App](http://localhost:8080) and 8081
 for the [StudyU Designer](http://localhost:8081).
 
-If you want to use a different database, more steps are necessary. You can start
-a managed Supabase project at [supabase.com](https://supabase.com) and link it
-with your StudyU instance by replacing the values for `STUDYU_SUPABASE_URL` and
-`STUDYU_SUPABASE_PUBLIC_ANON_KEY` in the file `.env.local`. Alternatively, you
-can self-host Supabase on your own as explained in
+By default the main StudyU database will be used. If you want to use a different
+database, more steps are necessary. You can start a managed Supabase project at
+[supabase.com](https://supabase.com) and link it with your StudyU instance by
+replacing the values for `STUDYU_SUPABASE_URL` and
+`STUDYU_SUPABASE_PUBLIC_ANON_KEY` in the file `flutter_common/lib/envs/.env.local`.
```

Comment:
I think for this use-case it'd be best to recommend using some git-ignored file instead of this one because the person doing this will otherwise have changes to the repo

Reply:
They'll have changes in the repo with the current setup as well when modifying ports in the docker compose scripts, customizing domains in the nginx config files, or switching the the login data in the .htpasswd file. This is far from optimal at the moment. Maybe those changes can be outsourced to the `docker/.studyu_config` file and injected with the cli at startup. But these are future todos. Having a repo with no changes is out of scope atm.

Reply:
Oki I'll add this to #499